yarn.lock

```diff
@@ -2767,11 +2767,6 @@ deep-equal@^1.0.1:
     object-keys "^1.1.1"
     regexp.prototype.flags "^1.2.0"
 
-deep-extend@^0.6.0:
```

Comment:
Did you expect all these changes in the yarn.lock for this PR?

Reply:
I'm pretty sure we do want these changes, but I did not mean to make them here. Moved to #3669